orion-server/src/main/java/com/pinterest/orion/core/automation/sensor/kafka/KafkaTopicSensor.java

```diff
@@ -142,4 +144,54 @@ private Map<String, KafkaTopicDescription> getTopicDescriptionFromKafka(KafkaClu
     return cluster.getTopicDescriptionFromKafka();
   }
 
+  public void populateTopicMetrics(KafkaCluster cluster, Map<String, KafkaTopicDescription> topicDescriptionMap) {
+    // Publish kafka topic metrics including topic size in byte, number of partitions and retention period in ms.
+    for (KafkaTopicDescription topicDescription : topicDescriptionMap.values()) {
+      Map<String, String> metricsTags = new HashMap<String, String>() {{
+        put("topicName", topicDescription.getName());
+        put("isInternal", String.valueOf(topicDescription.isInternal()));
+        put("clusterName", cluster.getName());
+        put("clusterId", cluster.getClusterId());
```

Comment:
Do you recall how `clusterName` and `clusterId` are different? I think we shouldn't need both.
Also, we can drop `isInternal` tag. Don't need to add unnecessary overhead on metric side.

Reply:
OK. Will remove those unnecessary tags. My initial thought was to add more tags so we can have more fields for filtering.